hass-addon-sunsynk-multi/README.md

```diff
@@ -10,3 +10,11 @@ See the docs for more info - [https://kellerza.github.io/sunsynk](https://keller
 Example of Home Assistant Energy Management, enabled by this addon.
 
 ![HASS Energy management](https://github.com/kellerza/sunsynk/raw/main/images/energy.png)
+
+## Usage of env vars
+
+You can also use env vars for config options. For them to work, you need to add one additional env var**: 
+
+S6_KEEP_ENV: 1
```

Comment:
Can you use env vars with addons?

Or would this be for the standalone deployments? might be best to document [here](https://kellerza.github.io/sunsynk/guide/standalone-deployment)

Reply:
Yes, the idea behind the PR is usage in standalone deployment.